test/quantization/fx/test_quantize_fx.py

```diff
@@ -6147,6 +6147,90 @@ def make_qconfig(scale, zp, dtype):
+                               padding=[0] * self.dim, dilation=[1] * self.dim, groups=1)
+
+        for dimension in [1, 2, 3]:
+            model = Mod(3, 16, 3, dimension)
+            model.eval()
+            qconfig = get_default_qconfig()
+            qconfig_mapping = QConfigMapping().set_global(qconfig)
+            input_shape = (1, 3, *([8] * dimension))
+            example_inputs = torch.randn(input_shape)
+            prepared_model = prepare_fx(model, qconfig_mapping, example_inputs)
+            prepared_model(example_inputs)
+            quantized_model = convert_fx(prepared_model)
+            # This should pass
+            quantized_model(example_inputs)
+
+    def test_lowering_functional_conv_transpose_with_kwargs(self):
+        dim_to_op = {
+            1: F.conv_transpose1d,
+            2: F.conv_transpose2d,
+            3: F.conv_transpose3d,
+        }
+
+        class Mod(nn.Module):
+            def __init__(self, in_channels, out_channels, kernel_size, dimension):
+                super().__init__()
+                self.dim = dimension
+                self.op = dim_to_op[dimension]
+                kernel_sizes = [kernel_size] * self.dim
+                self.weight = nn.Parameter(torch.randn(in_channels, out_channels, *kernel_sizes))
+
+            def forward(self, input):
+                return self.op(input, self.weight, bias=None, stride=[1] * self.dim,
+                               padding=[0] * self.dim, output_padding=[0] * self.dim,
+                               dilation=[1] * self.dim, groups=1)
+
+        for dimension in [1, 2, 3]:
+            model = Mod(3, 16, 3, dimension)
+            model.eval()
+            qconfig = get_default_qconfig()
+            qconfig_mapping = QConfigMapping().set_global(qconfig)
+            input_shape = (1, 3, *([8] * dimension))
+            example_inputs = torch.randn(input_shape)
+            prepared_model = prepare_fx(model, qconfig_mapping, example_inputs)
+            prepared_model(example_inputs)
+            quantized_model = convert_fx(prepared_model)
+            # This should pass
+            quantized_model(example_inputs)
+
+    def test_lowering_functional_linear_with_kwargs(self):
+        class Mod(nn.Module):
+            def __init__(self, in_channels, out_channels):
+                super().__init__()
+                self.weight = nn.Parameter(torch.randn(out_channels, in_channels))
+
+            def forward(self, input):
+                return F.linear(input, self.weight, bias=None)
+
+        model = Mod(8, 4)
+        model.eval()
+        qconfig = get_default_qconfig()
+        qconfig_mapping = QConfigMapping().set_global(qconfig)
+        example_inputs = torch.randn(1, 8)
+        prepared_model = prepare_fx(model, qconfig_mapping, example_inputs)
+        prepared_model(example_inputs)
+        quantized_model = convert_fx(prepared_model)
+        # This should pass
+        quantized_model(example_inputs)
```

Comment:
for all these tests, can we also test that the quantized model has the expected set of ops?

Reply:
Thanks for reviewing. I have added the check. Also, I have removed the test case for transposed conv since it is not yet implemented to lower functional transposed conv. This should be fixed in another PR.

Reply:
Hi @supriyar I have opened another two PRs to enable lowering transposed conv:
- https://github.com/pytorch/pytorch/pull/97125
- https://github.com/pytorch/pytorch/pull/97126

I think maybe we land this PR first. Then I will add test cases for transposed conv back in a separate PR for the kwargs issue. How does that sound to you? Thanks.

Reply:
yes, that sounds good to me.

Reply:
Thanks